### ISSUE TITLE
composition of Pad and Crop lead to a nifti header change

### ISSUE THREAD

**Issue by romainVala** [fepegar/torchio#351]
**🐛Bug**

Not sure if it is a bug, sorry if I miss something: I want to do a padding (before model prediction) and then a crop on the prediction, but I notice a change in the nifti header, which make it as if there was a translation (if I save the output in a nifti file )

**To reproduce**

```python
# Your code here
import torchio as tio
sub = tio.datasets.Colin27()

t = tio.Compose([tio.Pad(padding=10), tio.Crop(bounds_parameters=10)])
new_sub = t(sub)

print(new_sub.t1.affine)
print(sub.t1.affine)

```

```python-traceback
# Your errors here
```

**Expected behavior**
the transformed image should have the save affine no ?

**Actual behavior**

**TorchIO version**

**Comment by fepegar** [comment]
Yes. It looks like a bug! I'll take a look.

**Comment by fepegar** [comment]
The problem seems to be in `Crop`.

**Comment by fepegar** [comment]
Fixed in `v0.17.53`. Thanks for reporting!

**Comment by romainVala** [comment]
thanks for correcting !
